lib/matplotlib/backend_bases.py

```diff
@@ -2952,6 +2952,63 @@ def _zoom_pan_handler(self, event):
             elif event.name == "button_release_event":
                 self.release_zoom(event)
 
+    def _start_event_axes_pan(self, event, method="zoom"):
```

Comment:
```suggestion
    def _start_event_axes_interaction(self, event, method="zoom"):
```

I had a moment of "wait, this PR does not do zoom!"

Reply:
True... that's still an artefact from the initial implementation which also had an explicit `_start_event_zoom`... 
Now `_start_event_axes_interaction` is used everywhere!